security/app-layer-policy.md

```diff
@@ -1,12 +1,12 @@
 ---
-title: Enable Calico application layer policy for Istio
-description: Enable Calico application layer policy for Istio service mesh.
+title: Enforce network policy for Istio
+description: Enforce network policy for Istio service mesh including matching on HTTP methods and paths.
 canonical_url: '/security/app-layer-policy'
 ---
 
 ### Big picture
 
-Enable {{site.prodname}} application layer network policy in Istio service mesh.
+Enable {{site.prodname}} application layer network policy in Istio service mesh to enforce cluster security.
```

Comment:
I prefer the text from the other doc: "Calico integrates seamlessly with Istio to enforce network policy within the Istio service mesh".  (Most readers won't understand what we mean by "application layer network policy".)

Reply:
Fixed.